### PR TITLE
Introduce Hindi Translation for Chapter-1

### LINKED ISSUE #337: Add Hindi translations to Ch.1
Add Hindi translations to Ch.1

### PR DESCRIPTION
**Pull request by shaavan**
resolves #337

Adds translation for Chapter-1

### CODE REVIEW

**Comment by shaavan** [comment]
Looking good so far!

**Screenshot:**

image

**Comment by shaavan** [comment]
It's Ready!!

image

**Comment by shaavan** [comment]
@vinayaksh42

Please take a look through the introduced translations to cross-check the introduced translations!

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]
The translation for Hindi LGTM! Especially the homepage looks amazing!
Just a few things can be improved that I noticed on the /chapters page:
Screenshot 2023-04-03 at 9 24 32 PM
The link for the feedback seems to be breaking up, you can check out the English translation file (`en.ts`). Another thing that could be improved is converting the Heading `Chapters` and `Explore the mysteries of Satoshi and learn about Bitcoin along the way.` to Hindi.

File: i18n/locales/hi.ts [hunk 1]
```diff
@@ -0,0 +1,320 @@
...
+    subheading_one: 'एक अवतार चुनें',
+    subheading_two: 'अपने व्यक्तिगत कोड का बैकअप लें',
+    paragraph_two:
+      'सब तैयार? कोड कॉपी किया गया और बैक-अप किया गया? सुनिश्चित करें कि आप ऐसा करते हैं, क्योंकि यदि आप इसे खो देते हैं तो इसे पुनर्प्राप्त नहीं किया जा सकता है।',
+    confirm: 'मैंने अपना कोड कॉपी और बैक अप लिया है',
+  },
+  demo_disclaimer: {
+    title: 'यह एक डेमो है',
+    description:
+      'हम अभी भी इस परियोजना के प्रारंभिक चरण में हैं। अध्याय 1 कार्यात्मक है, शेष विकास के अधीन है। इसे आज़माएं और हमें बताएं कि आप क्या सोचते हैं, हमें <Link href="https://docs.google.com/forms/d/e/1FAIpQLSf1xpNqUYJyvYL5IZDnxy78273pkqzfYW2Hf91H4Do4KHgy9g/viewform?usp=sf_link" className="underline">आपकी प्रतिक्रिया</लिंक> अच्छी लगेगी।"',
```
Comment (vinayaksh42) [inline i18n/locales/hi.ts:307]: you accidentally wrote `</Link>` as `</लिंक>`

**Comment by shaavan** [comment]
Addressed @vinayaksh42 comment

- Fixed the <Link> tag on the `/chapters` page.

**Comment by benalleng** [comment]
I made a change in `/components/LangSwitch.tsx` in #379 so that we don't have to keep adding new languages to make it easier for future contributors

**Comment by shaavan** [comment]
Updated from 23b38204b7335541900d0ac3b9e71bdb42c7640d to cfbfd2d1eac61b53da6997d57f3c63493f63aeff ([pr345.02](https://github.com/shaavan/saving-satoshi/commits/pr345.02) -> [pr345.03](https://github.com/shaavan/saving-satoshi/commits/pr345.03))
Addressed @benalleng comment

**Changes:**
1. Rebased over master and resolved merge conflicts.
2. Introduced translation for chapter-2.

**Review by vinayaksh42 — CHANGES_REQUESTED** [review]
the PR seems to be having build issues, @shaavan can you check what might be causing it?

**Comment by shaavan** [comment]
I can compile my branch successfully on my local system using both `npm run build` and `npm run start`.

<details>
<summary>
<strong>Screenshot:</strong>
</summary>
image
</details>

@afterburn

Could you please check the possible issue with the Vercel built for this branch? Thanks!

**Comment by benalleng** [comment]
Build succeeded on my end too

**Comment by vinayaksh42** [comment]
not sure what might be causing the issue because the build seems to be running fine on my system as well. One thing that I noticed when I ran `npm run build` on this branch and the master branch is the size of `First load JS`, it seems to be double compared to the master branch.

** ScreenShots **
Master branch:
Screenshot 2023-04-18 at 1 14 04 PM

This branch:
Screenshot 2023-04-18 at 1 13 01 PM

**Comment by benalleng** [comment]
@shaavan I made a test build in #384 that succeeded, you might want to merge with master and see if there was some old state causing this build to fail

**Comment by shaavan** [comment]
Updated from cfbfd2d1eac61b53da6997d57f3c63493f63aeff to aef3850163dd5b790146392b2147925eb6fe87c0 ([pr345.03](https://github.com/shaavan/saving-satoshi/commits/pr345.03) -> [pr345.04](https://github.com/shaavan/saving-satoshi/commits/pr345.04))
Addressed @benalleng suggestion

Changes:
1. Rebased over master.

**Comment by ecurrencyhodler** [comment]
Success!

**Review by vinayaksh42 — APPROVED** [review]

**Pull request merged**